WordPress/src/main/java/org/wordpress/android/ui/mediapicker/loader/DeviceListBuilder.kt

```diff
@@ -103,8 +103,16 @@ class DeviceListBuilder(
         val deviceMediaList = deviceMediaLoader.loadMedia(mediaType, filter, pageSize, lastDateModified)
         val result = deviceMediaList.items.mapNotNull {
             val mimeType = deviceMediaLoader.getMimeType(it.uri)
-            if (mimeType != null && MediaUtils.isSupportedMimeType(mimeType) &&
-                    mediaUtilsWrapper.isMimeTypeSupportedBySitePlan(site, mimeType)) {
+            val isMimeTypeSupported = mimeType != null && site?.let {
```

Comment:
I ended up putting the condition-based logic into a variable because detekt was complaining that it was too complex.